docs/language/contracts.rst

```diff
@@ -94,15 +94,19 @@ __________________________________
 
 On Solana, the contract being created must have the ``@program_id()`` annotation that specifies the program account to
 which the contract code has been deployed. This account holds only the contract's executable binary.
-When calling a constructor, one needs to provide an address that will serve as the contract's data account,
-by using the call argument ``address``:
+When calling a constructor from an external function, no call arguments are required. The data account necessary to
```

Comment:
"When calling a constructor from an external function, no call arguments are required."

That's not always true, this needs a qualifier (not in the next paragraph).